### PR TITLE
CBG-745 - Set up for 2.7.2 builds

### PR DESCRIPTION
**Pull request by bbrks**

### CODE REVIEW

File: manifest/2.7.xml [hunk 1]
```diff
@@ -14,14 +14,14 @@
     <!-- Build Scripts (required on CI servers) -->
     <project name="product-texts" path="product-texts" remote="couchbase"/>
     <project name="build" path="cbbuild" remote="couchbase">
-        <annotation name="VERSION" value="2.7.1"     keep="true"/>
+        <annotation name="VERSION" value="2.7.2"     keep="true"/>
```
Comment (adamcfraser) [inline manifest/2.7.xml:17]: I think the build project should be pinned, based on the approach taken in 2.6.xml.

**Review by adamcfraser — CHANGES_REQUESTED** [review]

File: manifest/2.7.xml (hunk 1 above)
Comment (bbrks) [reply manifest/2.7.xml:17]: Pinning to the current HEAD of cbbuild

File: manifest/2.7.xml [hunk 2]
```diff
@@ -13,15 +13,15 @@
 
     <!-- Build Scripts (required on CI servers) -->
     <project name="product-texts" path="product-texts" remote="couchbase"/>
-    <project name="build" path="cbbuild" remote="couchbase">
-        <annotation name="VERSION" value="2.7.1"     keep="true"/>
+    <project name="build" path="cbbuild" remote="couchbase" revision="a08bf70a05fc5b94e62c6aa2d349d3f1e261f1cc">
```
Comment (adamcfraser) [inline manifest/2.7.xml:16]: Nice job catching that the release build (2.7.1-5) had a different build SHA than the latest on build-manifests/sync_gateway.

Should the build version in the 2.7.1.xml manifest be updated to the same thing?

**Review by adamcfraser — CHANGES_REQUESTED** [review]

File: manifest/2.7/2.7.1.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    <remote fetch="https://github.com/tleyden/" name="tleyden"/>
+
+    <remote fetch="ssh://git@github.com/couchbaselabs/" name="couchbaselabs_private" />
+
+    <default remote="couchbase" revision="master"/>
+
+    <!-- Build Scripts (required on CI servers) -->
+    <project name="product-texts" path="product-texts" remote="couchbase"/>
+    <project name="build" path="cbbuild" remote="couchbase" revision="6b173f045016bf510d6d2076c97fb54d7df01a69">
```
Comment (bbrks) [inline manifest/2.7/2.7.1.xml]:
```suggestion
    <project name="build" path="cbbuild" remote="couchbase" revision="a08bf70a05fc5b94e62c6aa2d349d3f1e261f1cc">
```

File: manifest/2.7.xml (hunk 2 above)
Comment (bbrks) [reply manifest/2.7.xml:16]: Heh, good catch. Updated.

**Review by adamcfraser — APPROVED** [review]

**Pull request merged**